shop_online_api/app/controllers/api/v1/orders_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Api::V1::OrdersController < ApplicationController
+  def index
+    @orders = Order.all
+    render json: @users, status: :ok
+  end
+
+  def show
+    @user = User.find(params[:id])
+    render json: @user
+  end
+
+  def create
+    confirm_token = request.headers['Access-token']
+    @user = User.find_by(confirm_token: confirm_token)
```

Comment:
- do we need `instance variable`?
- `find_by` If no record is found, returns nil. => update failed